src/hub/atlas.js

```diff
@@ -421,6 +421,79 @@ class AtlasClient {
         return users;
     }
 
+    /**
+     * Update an existing users data
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint PATCH method
+     */
+    async patchUser(data) {
+        const op = { method: "PATCH", body: { ...data } };
+        try {
+            const resp = await this.fetch("/v1/user/" + data.id, op);
+            if(!resp.ok) {
+                const e = new errors.ProtocolError(resp.status, respContent);
+                e.message = `${urn} (${text})`;
+                throw e;
+            }
+        } catch (err) {
+            throw err;
```

Comment:
just remove the whole try/catch block.  Let it destroy worlds. :smile:

Reply:
And I was wrong about the this.fetch() needing `resp.ok`, the AtlasClient.fetch already does that and returns the contents directly.  so I think this is just a one line function when condensed.  The above code is checking resp.ok which as it turns out isn't ever anything but undefined (or worse, a TypeError if resp is undefined)